Model/Product/Feed/Builder/Inventory.php

```diff
@@ -95,7 +95,8 @@ public function initInventoryForProduct(Product $product)
     public function getAvailability()
     {
         return $this->product && $this->productStock && $this->productStock->getIsInStock()
-        && ($this->productStock->getQty() - $this->systemConfig->getOutOfStockThreshold() > 0)
+        && ((($this->productStock->getQty() - $this->systemConfig->getOutOfStockThreshold()) > 0)
+            || !$this->productStock->getManageStock())
```

Comment:
@Bashev, can you please simplify this so it's easier to read? Also, we're already checking if stock is managed in `$this->productStock->getIsInStock()` let's not use a combination of both checks here.